sys-fs/cryfs/cryfs-9999.ebuild

```diff
@@ -40,6 +40,9 @@ src_prepare() {
 	# remove tests that require internet access to comply with Gentoo policy
 	sed -i -e '/CurlHttpClientTest.cpp/d' -e '/FakeHttpClientTest.cpp/d' test/cpp-utils/CMakeLists.txt || die
 
+	# remove non-applicable warning caused by cmake-utils setting NDEBUG
+	sed -i -e '/WARNING! This is a debug build. Performance might be slow./d' src/cryfs-cli/Cli.cpp || die
```

Comment:
Sounds like upstream is doing something really silly. Did you submit a bug and/or a patch removing this?